css/home.css

```diff
@@ -2,217 +1,170 @@
+body,
+html {
   margin: 0;
   padding: 0;
+  width: 100%;
   overflow-x: hidden;
-  font-family: fantasy;
 }
 
-header {
-  display: grid;
-  grid-template-columns: auto auto;
-  grid-template-rows: auto auto;
-}
-
-.logo {
-  display: grid;
-  grid-row: 1 / 3;
-  overflow: hidden;
+.head {
+  position: absolute;
 }
 
-.logo a {
-  justify-self: center;
-  align-self: center;
+.top-layer {
+  z-index: 2;
 }
 
-.nav-1 {
-  display: grid;
-  justify-content: end;
-  align-content: center;
-  grid-template-columns: auto auto auto auto;
-  grid-gap: 15px;
-  min-height: 50px;
+.dropdown-menu.head {
+  transform: translateY(-5px);
 }
 
-.nav-1 a {
-  text-decoration: none;
+.flag-icon {
+  height: 20px;
 }
 
-.nav-1 a img {
-  width: 20px;
+.bg-custom {
+  transform: translateY(20px);
+  background: transparent !important;
+  transition: all 700ms;
 }
 
-.nav-2 {
-  display: grid;
-  min-height: 50px;
-  grid-auto-flow: column;
-  grid-gap: 20px;
+.bg-custom.scrolled {
+  transform: translateY(0px);
+  background: white !important;
 }
 
-.navlink {
-  display: grid;
-  justify-content: end;
-  align-content: center;
-  grid-auto-flow: column;
-  grid-gap: 10px;
+.nav-link {
+  color: white;
+  transition: 500ms;
 }
 
-.dropdown-content {
-  font-size: small;
-  text-align: center;
-  width: 200px;
-  text-overflow: clip;
-  display: grid;
-  position: absolute;
-  grid-template-columns: 1fr;
-  margin-top: 5px;
-  transform-origin: top;
-  transform: scaleY(0);
-  transition: all 300ms;
+.nav-link:hover {
+  background: #eaba3a;
+  color: white;
 }
 
-.dropdown:hover .dropdown-content {
-  transform: scaleY(1);
+.nav-link.scrolled {
+  color: rgb(75, 75, 75);
 }
 
-.navlink a {
-  padding: 8px;
-  background-color: #fff;
-  text-decoration: none;
-  transition: 300ms;
-  border-radius: 2px;
+.dropdown-menu {
+  transform: translate(-30%, -5px);
 }
 
-.navlink a:hover {
-  background-color: #eaba3a;
-  color: #fff;
+.dropdown-menu a:hover {
+  background: #eaba3a;
 }
 
-.ceo {
-  padding-left: 20%;
+.dropdown:hover>.dropdown-menu {
+  display: block;
 }
 
-.sambutan {
-  display: grid;
-  grid-auto-flow: column;
-  grid-gap: 10%;
-  background-color: lightgray;
+.dropdown>.dropdown-toggle:active {
+  /*Without this, clicking will make it sticky*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)